docs/tutorials/python/linear-regression.md

```diff
@@ -94,32 +89,31 @@ lro = mx.sym.LinearRegressionOutput(data=fully_connected_layer, label=Y, name="l
 
 The above network uses the following layers:
 
-1. FullyConnected: The fully connected symbol represents a fully connected layer
+1. `FullyConnected`: The fully connected symbol represents a fully connected layer
    of a neural network (without any activation being applied), which in essence,
    is just a linear regression on the input attributes. It takes the following
    parameters:
 
-   - data: Input to the layer (specify the symbol whose output should be fed here)
-   - num_hidden: Number of hidden dimension which specifies the size of the output of the layer
+   - `data`: Input to the layer (specifies the symbol whose output should be fed here)
+   - `num_hidden`: Number of hidden dimensions which specifies the size of the output of the layer
 
-2. Linear Regression Output: Output layers in MXNet aim at implementing a
-   loss. In our example, the Linear Regression Output layer is used which
-   specifies that an l2 loss needs to be applied against it's input and the
-   actual labels provided to this layer. The parameters to this layer are:
+2. `LinearRegressionOutput`: Output layers in MXNet aim to implement a
```

Comment:
"Output layers in MXNet aim to implement a loss"

Layers don't aim to do anything. Please reword. What precisely is meant to be said here?